### PR TITLE
Initial checkin of the webapp

### PR DESCRIPTION
**Pull request by brianjjones**

### CODE REVIEW

File: pix2pix/pix2pix_website/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## **The flow:**
+- Upon clicking 'Render' on the Draw page or selecting an image on the Image Select page, the webapp will bring up a pending result element. The result will show the starting image and a 'Working' animation while it waits for the result from the server.
+- A POST '/render' call is made, which app.js will handle. For the Draw page an image buffer from the canvas element is sent, for the Image Select page a image URL is sent.
+- app.js will then call the function **uploadToPix2Pix(img)**, which will use the provided REST API to send an image to the FaaS server for rendering.
+- FaaS server will return the rendered image, and app.js will then finish the reply to the POST '/render' call. This reply will contain the request ID, the starting image path, and the rendered image path.
+- The webapp will get the reply and will disply the result on the main page and add the result to the history. Note that only 12 items are stored in the history, and will be overwritten as the demo is run more than 12 times. This number is arbitrary and can be increased or decreased if desired.
+
+# TODO
```
Comment (DnPlas) [inline pix2pix/pix2pix_website/README.md]: Not sure if we want to add a ToDo section. Thoughts?

Comment (unrahul) [reply pix2pix/pix2pix_website/README.md]: +1 most of the TODO section looks like already completed, we can remove it @brianjjones  what do u say?

File: pix2pix/pix2pix_website/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Pix 2 Pix demo
+
+A demonstration of Deep Learning and FaaS
+
+## Installation
+From the pix2pix_website directory, install the required npm packages.
+```
+npm install
+```
+# Usage
```
Comment (DnPlas) [inline pix2pix/pix2pix_website/README.md]: Can we add a "pre requisites" section specifying you need an openfaas deployment?

File: pix2pix/pix2pix_website/README.md (hunk 1 above)
Comment (brianjjones) [reply pix2pix/pix2pix_website/README.md]: Whoops! Yeah it can come out.

File: pix2pix/pix2pix_website/README.md (hunk 2 above)
Comment (brianjjones) [reply pix2pix/pix2pix_website/README.md]: Sound good, added.

File: pix2pix/pix2pix_website/README.md (hunk 1 above)
Comment (brianjjones) [reply pix2pix/pix2pix_website/README.md]: Alright, TODO section removed.

**Comment by unrahul** [comment]
+2 thanks @brianjjones , merging this

**Pull request merged**